src/Api/Controllers/AppsSettingsController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Canvas\Api\Controllers;
+
+use Canvas\Models\Apps;
+use Canvas\Dto\AppsSettings;
+use Phalcon\Http\Response;
+
+/**
+ * Class LanguagesController.
+ *
+ * @package Canvas\Api\Controllers
+ *
+ */
+class AppsSettingsController extends BaseController
```

Comment:
@rwhite27 the method create, delete, index , declare them and return empty since bakacontroller has it we can expose it